src/state/nftMarket/hooks.ts

```diff
@@ -4,18 +4,24 @@ import { useAppDispatch } from 'state'
 import { pancakeBunniesAddress } from 'views/Nft/market/constants'
 import { isAddress } from 'utils'
 import useRefresh from 'hooks/useRefresh'
-import { fetchCollections, fetchNftsByBunnyId, fetchNftsFromCollections, updateNftTokensData } from './reducer'
+import { fetchCollection, fetchCollections, fetchNftsByBunnyId, updateNftTokensData } from './reducer'
 import { State } from '../types'
 import { NftToken, UserNftsState } from './types'
 
 export const useFetchCollections = () => {
   const dispatch = useAppDispatch()
   useEffect(() => {
     dispatch(fetchCollections())
-    dispatch(fetchNftsFromCollections(pancakeBunniesAddress))
```

Comment:
There's a reliance on this in "More from this collection" section.
Don't merge it yet. I'll refactor that bit and then this can go